.github/workflows/ci.yml

```
@@ -238,11 +238,23 @@ jobs:
           checks/requirements.py
 
   notebook:
-    name: Notebook check
     env:
-      SUPERSTAQ_API_KEY : ${{ secrets.SUPERSTAQ_API_KEY }}
-      IBMQ_TOKEN : ${{ secrets.IBMQ_TOKEN }}
+      SUPERSTAQ_API_KEY: ${{ secrets.SUPERSTAQ_API_KEY }}
+      IBMQ_TOKEN: ${{ secrets.IBMQ_TOKEN }}
     runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        include:
+          - type: "qiskit-superstaq"
+            pattern: "*_qss.ipynb"
```

notebooks from qiskit-superstaq/ should probably be included here as well
```suggestion
            pattern: "*_qss.ipynb qiskit-superstaq"
```